bundles/org.openhab.binding.ecobee/src/main/java/org/openhab/binding/ecobee/internal/handler/EcobeeUtils.java

```diff
@@ -110,7 +110,7 @@ public static Integer convertQuantityTypeToEcobeeTemp(Object value) {
             QuantityType<Temperature> convertedTemp = ((QuantityType<Temperature>) value)
                     .toUnit(ImperialUnits.FAHRENHEIT);
             if (convertedTemp != null) {
-                return Integer.valueOf(convertedTemp.intValue() * 10);
+                return Integer.valueOf((int)(convertedTemp.doubleValue() * 10));
```

Comment:
Yes, since the Ecobee API supports tenths, it's a good idea to take advantage of it.

But, this always will truncate (i.e. round down), correct? So, for example, 70.999 will convert to 709. Is that what we want, or would we rather it round up or down (to 710 in my example)? Just wondering why you chose to truncate vs round up/down.

Reply:
Yes, clearly it was wrong for the origin code to truncate to whole degrees. 😉 

And for C values of .0 and .5 (which I suspect are the most likely), truncating will work perfectly fine.